app/services/markdown_processor/parser.rb

```diff
@@ -35,7 +37,8 @@ def finalize(link_attributes: {})
       code_tag_content = convert_code_tags_to_triple_backticks(@content)
       escaped_content = escape_liquid_tags_in_codeblock(code_tag_content)
       html = markdown.render(escaped_content)
-      sanitized_content = sanitize_rendered_markdown(html)
+      sanitized_content = ActionController::Base.helpers.sanitize html, @sanitize_options
```

Comment:
I used the method explicitly here, it's more conveniet to pass options, and it doesn't make sense to include the whole helper to use 1 method.